recipes/ibm_db_sa/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+cudnn:                                    # [linux64]
+  - undefined                             # [linux64]
+cuda_compiler_version:                    # [linux64]
+  - None                                  # [linux64]
+docker_image:                             # [linux64]
+  - condaforge/linux-anvil-cos7-x86_64    # [linux64]
+cdt_name:                                 # [linux64]
+  - cos7                                  # [linux64]
```

Comment:
This is not needed, you can add the following to `conda-forge.yml` on the feedstock:

```
os_version:
  linux_64: cos7
```